config/app.default.php

```diff
@@ -244,6 +249,8 @@
              * which is the recommended value in production environments
              */
             //'init' => ['SET GLOBAL innodb_stats_on_metadata = 0'],
+            
+            'url' => env('DATABASE_URL', ''),
```

Comment:
@josegonzalez 
Nothing special. I just set `DATABASE_URL`, without setting `DATABASE_TEST_URL`, then I got the warning and CakePHP can not connect.